module/metrics.go

```diff
@@ -94,13 +94,13 @@ type CleanerMetrics interface {
 }
 
 type CacheMetrics interface {
-	// report the total number of cached items
+	// CacheEntries report the total number of cached items
 	CacheEntries(resource string, entries uint)
-	// report the number of times the queried item is found in the cache
+	// CacheHit report the number of times the queried item is found in the cache
 	CacheHit(resource string)
-	// report the number of items the queried item is not found in the cache, nor found in the database
+	// CacheNotFound report the number of items the queried item is not found in the cache, nor found in the database
 	CacheNotFound(resource string)
-	// report the number of items the queried item is not found in the cache, but found in the database
+	// CacheMiss report the number of items the queried item is not found in the cache, but found in the database
```

Comment:
s~(number of) items~$1 times~ ? :-)

Reply:
This is not within my context, I just fixed a `godoc` to be complaint with Golang best practices, i.e., `godoc` must start with the name of its corresponding program entity. In this case, it must start with `CacheMiss`. That is the only change it is applying. Any other comment is better consulted with the original developer.

Reply:
FYI I'm only suggesting that there is a simple typo :-)
```
// CacheMiss report the number of items the queried item is not found in the cache, but found in the database
// CacheMiss report the number of times the queried item is not found in the cache, but found in the database
```

Reply:
yes, correct, should be `number of times`.

Reply:
fixed in https://github.com/onflow/flow-go/pull/2093/commits/be72fd8857db97c13673034623438c30895ac6fd.